main.go

```diff
@@ -110,6 +111,20 @@ func main() {
 			DefaultText: "0, ie profiling disabled",
 			EnvVars:     []string{"BAZEL_REMOTE_PROFILE_PORT"},
 		},
+		&cli.IntFlag{
```

Comment:
Should we use `DurationFlag`'s for these, like we do for `--idle_timeout`?

Reply:
Good point, using Duration now.